docs/server/Templates/_WorkingWith.mdx

```diff
@@ -0,0 +1,31 @@
+import AsyncDisclaimer from "./_AsyncDisclaimer.mdx";
+
+## Working with the SDK
+
+### Checking a Gate
```

Comment:
hmm this is interesting,  It sort of combines the guide with the sdk docs. Not totally sold on that but this is great for now